### PR TITLE
Deprecate some more (bool, bool, bool) orientation functions

### PR DESCRIPTION
**Pull request by drwells**
Part of #14667.

### CODE REVIEW

File: include/deal.II/base/qprojector.h [hunk 1]
```diff
@@ -310,10 +316,32 @@ class QProjector
          const bool           face_rotation,
          const unsigned int   n_quadrature_points);
 
+    /**
+     * Static function to generate an offset object for a given face of a cell
+     * with the given combined face orientation. This function of course is only
+     * allowed if <tt>dim>=2</tt>, and the orientation, is ignored if the space
```
Comment (kronbichler) [inline include/deal.II/base/qprojector.h:322]:
```suggestion
     * allowed if <tt>dim>=2</tt>, and the orientation is ignored if the space
```

File: include/deal.II/base/qprojector.h [hunk 2]
```diff
@@ -322,6 +350,16 @@ class QProjector
          const bool                      face_rotation,
          const hp::QCollection<dim - 1> &quadrature);
 
+    /**
+     * Like the above function but taking a quadrature collection, enabling
+     * that each face might have different number of quadrature points.
```
Comment (kronbichler) [inline include/deal.II/base/qprojector.h:355]: I am not entirely sure, but would
```suggestion
     * that each face might have a different number of quadrature points.
```
be slightly more accurate?

**Review by kronbichler — APPROVED** [review]
This looks good to me.

File: include/deal.II/base/qprojector.h (hunk 2 above)
Comment (drwells) [reply include/deal.II/base/qprojector.h]: You're right. I find the comments of these functions pretty hard to understand - I'll add rewriting them to the orientation TODO list.

For now I rewrote this comment and the equivalent one for the deprecated version.

**Review by masterleinad — APPROVED** [review]

**Review by kronbichler — APPROVED** [review]

**Pull request merged**